en-US/main.lang

```diff
@@ -869,3 +869,13 @@ Subscribe to monthly updates and keep current with Mozilla news, including the l
 
 ;Get Firefox tips, tricks, news and more
 Get Firefox tips, tricks, news and more
+
+
+# This  string is a potential navigation label on Technology page
+;Innovations
+Innovations
```

Comment:
We need to drop this string, I just remembered that I asked to only use "Web Innovations"
https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/technology.html#L40